pyproject.toml

```
@@ -51,6 +51,16 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+default = [
+    "brotli; implementation_name=='cpython'",
+    "brotlicffi; implementation_name!='cpython'",
+    "certifi",
+    "mutagen",
+    "pycryptodomex",
+    "requests>=2.31.0,<3",
+    "urllib3>=1.26.17,<3",
+    "websockets>=12.0",
+]
```

```suggestion
default = []
```

Wouldn't this be fine? Why maintain a copy? We can add `curl_cffi` to this list and not to `dependencies`. Then move everything over in a few versions

---

Yeah, I guess I copied everything over for demonstrative purposes, but this makes more sense.

But I think `curl_cffi` should still be kept in its own optional dependency group; the devs over there have discovered that building wheels on Linux is a major problem.